### PR TITLE
chore: use checkbox indicator to locate checkboxes on login, click objects in the middle

### PR DESCRIPTION
**Pull request by anastasiyaig**
### What does the PR do

- click checkboxes' indicators on `Before get started` modal instead of checkbox item (which has text as well)
- click other elements in the middle by default
- remove `:` to make windows happy

### Affected areas

ui tests

### CODE REVIEW

**Review by Valentina1133 — APPROVED** [review]

File: test/e2e/gui/elements/check_box.py [hunk 1]
```diff
@@ -14,7 +14,7 @@ class CheckBox(QObject):
     @allure.step("Set {0} value: {1}")
     def set(self, value: bool):
         if self.is_checked is not value:
-            self.click()
+            self.click(x=5, y=5)
```
Comment (alexjba) [inline test/e2e/gui/elements/check_box.py:17]: wouldn't self.click(x=width * 0.5, y=height * 0.5) be safer? Not sure what x=5;y=5 would mean for every control.

Comment (anastasiyaig) [reply test/e2e/gui/elements/check_box.py:17]: it is not for every control. it is only for checkbox that will be clicked with (width=5, height=5), it is done specifically to click checkbox on onboarding screen where checkbox has Terms and Conditions link. So when i click that checkbox in the middle, it clicks the link instead @alexjba

Comment (anastasiyaig) [reply test/e2e/gui/elements/check_box.py:17]: i will do a click for indicator instead for checkboxes , thanks for hints :)

```
--- ui/StatusQ/src/StatusQ/Controls/StatusCheckBox.qml
+++ ui/StatusQ/src/StatusQ/Controls/StatusCheckBox.qml
@@ -43,6 +43,7 @@ CheckBox {
     font.pixelSize: size === StatusCheckBox.Size.Regular ? 15 : 13
 
     indicator: Rectangle {
+        objectName: "indicator"
         anchors.left: root.leftSide? parent.left : undefined
         anchors.right: !root.leftSide? parent.right : undefined
         implicitWidth: size === StatusCheckBox.Size.Regular
```

File: test/e2e/gui/objects_map/names.py [hunk 2]
```diff
@@ -43,7 +43,9 @@
 """ Onboarding """
 # Before you get started Popup
 acknowledge_checkbox = {"checkable": True, "container": statusDesktop_mainWindow_overlay, "objectName": "acknowledgeCheckBox", "type": "StatusCheckBox", "visible": True}
+acknowledgeIndicator = {"container": acknowledge_checkbox, "objectName": "indicator", "type": "Rectangle", "visible": True}
```
Comment (caybro) [inline test/e2e/gui/objects_map/names.py:46]: Do you have to specify the `Rectangle` type here? It is like that but could be "anything". I'd go with `Item` if you really have to

**Review by caybro — APPROVED** [review]

File: test/e2e/gui/objects_map/names.py (hunk 2 above)
Comment (anastasiyaig) [reply test/e2e/gui/objects_map/names.py:46]: ah good point, i can remove the type completely, i dont really rely on it

**Comment by jakubgs** [comment]
The E2E job has correct parameters:
image

We can see the correct version `8d807221106c88a21fc06dedf7d54e4dc9190655` was checked out by E2E job:
```
 > git fetch --tags --force --progress -- https://github.com/status-im/status-desktop +refs/heads/*:refs/remotes/origin/* # timeout=10
Avoid second fetch
Checking out Revision 8d807221106c88a21fc06dedf7d54e4dc9190655 (detached)
Commit message: "chore: click indicator on before get started modal instead of checkboxes"
```
https://ci.status.im/job/status-desktop/job/e2e/job/prs/292/console

Same as the one used by the Linux build:
```
 > git fetch --no-tags --force --progress -- https://github.com/status-im/status-desktop.git +refs/pull/15177/head:refs/remotes/origin/PR-15177 # timeout=10
Checking out Revision 8d807221106c88a21fc06dedf7d54e4dc9190655 (PR-15177)
Commit message: "chore: click indicator on before get started modal instead of checkboxes"
```
https://ci.status.im/job/status-desktop/job/prs/job/linux/job/x86_64/job/package/job/PR-15177/6/console

So I see no issue here.

**Comment by anastasiyaig** [comment]
@Valentina1133 i tested it one more time, the only test failed here is rename keypair which failed because of the application was not able to check second checkbox. It was able to click it everywhere in all the remaining tests. I will think of how to improve this, so we wont click Get started button until both checkboxes are checked. I think we can merge this now and monitor in master

https://ci.status.im/job/status-desktop/job/e2e/job/manual/2101/allure/#suites/99878b742645b26873fccfcdfe1b2095/f2c67dc3129d3580/

**Pull request merged**